hometask_18.py

```diff
@@ -0,0 +1,43 @@
+class Bot:
+    def init(self, name):
+        self.name = name
+
+    def say_name (self):
+        print(self.name)
+
+    def send_message (self, message):
+        print(message)
+
+class TelegramBot(Bot):
+
+    def set_url(self, url = None):
+        self.url = url
+
+    def set_chat_id(self, chat_id = None):
+        self.chat_id = chat_id
+
+    def send_message(self, message = None):
+        try:
```

Comment:
Why do you need `a` and `b` variables here? And why do you call set_url and set_chat_id here?